CONTRIBUTING.md

```diff
@@ -292,17 +292,14 @@ To consume the a dev package set the exact version in your project or to consume
 
 ## Preparing to a release of the new library
 
-To update the CHANGELOG, version and release tracking information use the `.\eng\scripts\Prepare-Release.ps1` script.
-
-The syntax is `.\eng\scripts\Prepare-Release.ps1 <package_name>`. The script would ask you for a new version or `NA` if you are not releasing in this cycle.
+To prepare a package for release you should make use of `.\eng\common\scripts\Prepare-Release.ps1` script passing it appropriate arguments for the package intended for release as well as the release date. This script will correctly update the package version in the repo as well as update DevOps release work items for that release. 
 
 If you are releasing out-of-band please use the `-ReleaseDate` parameter to specify the release data. `ReleaseDate` should be in `yyyy-MM-dd` format.
 
 Example invocations:
 
 ```powershell
-.\eng\scripts\Prepare-Release.ps1 Azure.Core
-.\eng\scripts\Prepare-Release.ps1 Azure.Core -ReleaseDate 2020-10-01
+.\eng\scripts\Prepare-Release.ps1 -PackageName "Azure.Core" -SerivceDirectory "core" -ReleaseDate "2020-10-01"
```

Comment:
To make this more simple we don't require "-PackageName" or "-ServiceDirectory" passed so I'd just remove those.

Reply:
I know we talked about this but if someone doesn't pass ServiceDirectory will the command still work as is?

Reply:
Nope, doesn't:
```
 .\eng\scripts\Prepare-Release.ps1 core
Source directory sdk

Assuming release is in February with release date 2021-02-09
Get-Content: D:\github\azure\net\eng\scripts\Prepare-Release.ps1:174
Line |
 174 |  … Version = ([xml](Get-Content "$packageDirectory/src/*.csproj")).Proje …
     |                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find path 'D:\github\azure\net\sdk\core\src' because it does not exist.


Latest released version 1.0.1, library type GA
```

Reply:
It does.

Reply:
But the issue matching doesn't seem to work:

```
.\eng\scripts\Prepare-Release.ps1 Azure.core
Source directory core

Assuming release is in February with release date 2021-02-09

Latest released version 1.8.1, library type GA
Input the new version, NA if you are not releasing, or press Enter to use use current project version '1.9.0':

Detected released type Minor

Updating versions to 1.9.0 with date 2021-02-09
Current Version: 1.9.0
WARNING: Version [1.9.0] is already present in change log with specificed ReleaseStatus [(2021-02-09)]. No Change made.
New Version: 1.9.0

The following work items exist:
Input the work item ID or press Enter to use ' - ' (fuzzy matched based on title):
```

Reply:
I'm dumb and am running the wrong script.

Reply:
Using only the package name doesn't work:
```
.\eng\common\scripts\Prepare-Release.ps1 Azure.Core
Get-PkgProperties: D:\github\azure\net\eng\common\scripts\Prepare-Release.ps1:30
Line |
  30 |  … perties -PackageName $PackageName -ServiceDirectory $serviceDirectory
     |                                                        ~~~~~~~~~~~~~~~~~
     | Cannot bind argument to parameter 'ServiceDirectory' because it is an empty string.

Source directory [  ]

Assuming release is in February with release date 02/09/2021
Input the new version, or press Enter to use use current project version '':
```

Reply:
Ok. @chidozieononiwu is going to make some changes to make that optional.